.github/workflows/benchmark_db_import.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Triggers a cloud function that imports the benchmark results from the GCS bucket into BigQuery
+# It is designed to be called from a parent workflow. Make sure to call it after all benchmark
+# related jobs have finished.
+
+name: Import Benchmark Results into DB
+
+on:
+  workflow_call:
+    inputs:
+      runner-group:
+        required: true
+        type: string
+      runner-env:
+        required: true
+        type: string
+
+env:
+  # TODO(beckerhe): Move this URL into GCE instance attributes
+  CLOUD_FUNCTION_URL: ${{ github.event_name == 'pull_request' && 'https://oobi-presubmit-importer-zbhz5clunq-uc.a.run.app' || 'https://oobi-postsubmit-importer-zbhz5clunq-uc.a.run.app' }}
+
+jobs:
+  call_cloud_function:
+    # This needs to run on a self-hosted runner to get access to the runner's service account
+    runs-on:
+      - self-hosted # must come first
+      - runner-group=${{ inputs.runner-group }}
+      - environment=${{ inputs.runner-env }}
+      - cpu
+      - os-family=Linux
+    steps:
+      - id: send_http_request
+        name: "Send HTTP request"
+        env:
+          GET_ID_TOKEN_URL: http://metadata/computeMetadata/v1/instance/service-accounts/default/identity?audience=${{env.CLOUD_FUNCTION_URL}}&format=full
+        continue-on-error: true
+        run: |
+          curl -sSLX POST \
```

Comment:
Can gcloud (https://cloud.google.com/functions/docs/running/direct) fetch the credential automatically on VM and call the cloud function? Like how we call `gcloud storage ...`

Reply:
Hmm. Good question - and good thinking. I need to try that.

Reply:
That works, but it requires the service account to have the cloudfunctions/call permission which is only part of the "Cloud Functions Developer" role.

I think I would prefer not to mess with the standard role assignments and only assign the "Cloud Functions Invoker" role to the service accounts here.